### PR TITLE
Add #ModelGarden MoViNet Video Clip Classification Notebook

### PR DESCRIPTION
**Pull request by KCFindstr**
This notebook covers the training and serving of the open source MoViNet model on clip classification.

**REQUIRED:** Fill out the below checklists or remove if irrelevant

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [x] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

### CODE REVIEW

File: notebooks/community/model_garden/model_garden_movinet_clip_classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,871 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_movinet_clip_classification.ipynb:1]: <h1>Video Clip Classification </h1><h1>-&gt;</h1><h1>video clip classification </h1><h1><br></h1><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_movinet_clip_classification.ipynb:1]: <p>I think you mean common helper functions.</p><p>Add some text explaining what each function does</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_movinet_clip_classification.ipynb:1]: <p>Training Dockers -&gt; training docker images</p><p>Docker -&gt; docker (lowercase when referring to the image, aka not the product)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_movinet_clip_classification.ipynb:1]: <p>Note that Bayesian optimization is the Vertex AI Vizier service</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_movinet_clip_classification.ipynb:1]: <p>model registry -&gt; the Vertex AI Model Registry</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_movinet_clip_classification.ipynb:1]: <p>Vertex -&gt; Vertex AI Model Registry</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_movinet_clip_classification.ipynb:1]: <p>on Vertex AI -&gt; with Vertex AI Batch Prediction</p><p>GCS -&gt; Cloud Storage</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (andrewferlitsch) [inline notebooks/community/model_garden/model_garden_movinet_clip_classification.ipynb:1]: <p>We -&gt; you</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Comment (KCFindstr) [reply notebooks/community/model_garden/model_garden_movinet_clip_classification.ipynb:1]: <p>Thanks for the review! Could you please clarify a little bit more on this comment - do you suggest I should mention Vertex AI Vizier service here? Aren't all search algorithms backed by Vertex AI Vizier?</p>

Comment (andrewferlitsch) [reply notebooks/community/model_garden/model_garden_movinet_clip_classification.ipynb:1]: You're correct, the underlying search service for all algorithms is Vizier

**Pull request merged**